### DISCUSSION TITLE
IRNAS SWL2001-Zephyr with SX1280 transceiver

### DISCUSSION THREAD

**Discussion by sluissantos** [zephyrproject-rtos/zephyr#77395] [General]
I need to use the LoRa Basic Modem with the SX1280 on Zephyr. I'm considering adapting the IRNAS SWL2001-Zephyr application, which currently uses the LR11xx, to work with the SX1280.
Could someone advise if this is a good approach? What challenges and technical considerations should I be aware of in this process?
Thanks.

**Comment by nordicjm** [comment]
https://github.com/zephyrproject-rtos/zephyr/discussions/72442
